src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -296,14 +296,14 @@ public static CodeAction Create(string title, Func<CancellationToken, Task<Solut
         /// <summary>
         /// Creates a top level code action with multiple code action choices a user can invoke.
         /// </summary>
-        public static CodeAction Create(string title, IEnumerable<CodeAction> nestedActions, string equivalenceKey = null)
+        public static CodeAction Create(string title, IEnumerable<CodeAction> nestedActions)
         {
             if (title == null)
             {
                 throw new ArgumentNullException(nameof(title));
             }
 
-            return new SimpleCodeAction(title, nestedActions.AsImmutableOrEmpty(), equivalenceKey);
+            return new SimpleCodeAction(title, nestedActions.AsImmutableOrEmpty(), equivalenceKey: string.Empty);
```

Comment:
This will cause all such top level/aggregate code actions to have identical equivalence key, which is not right. For example, if you had 2 suppressable diagnostics with same ID on the same line, the top level suppression code actions need to have different equivalence keys to ensure both of them are shown. However, if both of them have the same ID, then we want to get them de-duplicated to show only one.

I feel equivalence key should always be non-null and non-empty for top level actions, and the default value should ensure that 2 top level actions which have "sequence equal" nested actions, have identical equivalence key. Instead of a default value of string.Empty, can we move the method you removed above (`SuppressionCodeAction .ComputeEquivalenceKey`) down here and use that to compute the default value? I am fine with you tweaking the implementation of that method, as long as we get the desired semantics.